tests/integration/conftest.py

```diff
@@ -1291,7 +1354,7 @@ def search_qtest(qtest_api_token, qtest_project_id, object_type, query, fields=N
         object_type (str): The type of qTest objects to search.
             (Valid values: 'requirements', 'test-cases', 'test-runs', 'defects')
         query (str): The query to execute against the given object type.
-        fields (list(str)): A list of qTest object field names to capture in the return data.
+        fields ([str]): A list of qTest object field names to capture in the return data.
```

Comment:
The JetBrains IDE does not recognize "list(*)" as a valid docstring data type. Fixed.